Sources/Sharing/SharedKeys/FileStorageKey.swift

```diff
@@ -353,8 +358,8 @@
           close(source.handle)
         }
       },
-      load: { try Data(contentsOf: $0) },
-      save: { try $0.write(to: $1) }
+      load: { url in try Data(contentsOf: url) },
+      save: { data, url in try data.write(to: url) }
```

Comment:
Would `data.write(to: url, options: .atomic)` work now?

Reply:
Oops, that was there but must have been dropped while cleaning up. Will bring back...

Reply:
Also thanks for using named arguments in these closures, makes code a lot easier to read. (With the currently limited support for column breakpoints, I wouldn't mind if newlines were sprinkled here and there as well, makes debugging a lot easier at times!)

Reply:
Haha sure thing. Just pushed some newlines. And the atomic write is now back up!